### DISCUSSION TITLE
Additional image fields not passed

### DISCUSSION THREAD

**Discussion by codedge** [spatie/statamic-responsive-images#169] [General]
In my containers blueprint for images I defined an additional field called _label_.

<img width="1652" alt="2022-08-29 at 09 11 36@2x" src="https://user-images.githubusercontent.com/4409904/187144229-6c4dd0df-faca-41c3-ad89-9ce07f3a81e7.png">

When I want to use this new _label_ filed in the template I cannot access it. I try it like this:

```html
{{ responsive:header_image ratio="21/9" quality:webp="80" glide:width="3200" title="{label}" }}
```

The generated html contains the _alt_ attribute, but not the label.

![2022-08-29 at 09 15 19@2x](https://user-images.githubusercontent.com/4409904/187144971-d5c14afc-9747-41f5-acff-56e2eb22d7a5.png)

Anything I am doing wrong or is this a bug?

Lates version Statamic and latest version 2.13.1 of this extension here.

**Comment by ncla** [comment]
Where exactly is `{label}` coming from in your Antlers front-end template though? This addon does not augment the custom asset blueprint values in a fancy way or make them available. You would have to do something like this yourself to make label available to `responsive` tag (not tested, just theoretically something like this should work):

```antlers
{{ asset url="/header_image.png" }}
  {{ responsive:header_image ratio="21/9" quality:webp="80" glide:width="3200" title="{label}" }}
{{ /asset }}
```

Alternatively you could try publishing the blade.php templates into your project and update `responsiveImage.blade.php` and add `title="{{ $asset['label'] }}"` attribute to `img`. Again I have not tested it, just writing in theory.
